src/Knp/DictionaryBundle/Dictionary/SimpleDictionary.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Knp\DictionaryBundle\Dictionary;
+
+use Knp\DictionaryBundle\Dictionary as DictionaryInterface;
+
+class SimpleDictionary implements DictionaryInterface
+{
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var mixed[]|\ArrayAccess
+     */
+    private $values;
+
+    /**
+     * @param string               $name
+     * @param mixed[]|\ArrayAccess $values
+     */
+    public function __construct($name, $values)
+    {
+        $this->name   = $name;
+        $this->values = $values;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getValues()
```

Comment:
I think the current API is a bit inconsistent:
```php
<?php
$dict = new SimpleDictionary('name', ['A' => 'first', 'B' => 'second']);
$dict->getValues(); // ['A' => 'first', 'B' => 'second']
arrray_values($dict->getValues()); // ['first', 'second']
$dict->getKeys(); // ['A', 'B']
```

IMHO, it would be better to have three distinct methods:
* `getKeyValues` (or something like that, maybe `getChoices` but seems to be too much tied to form language)
* `getValues`
* `getKeys`